PubNub/Network/PNURLBuilder.m

```diff
@@ -16,7 +16,7 @@
  @since 4.0
  */
 static NSString * const PNOperationRequestTemplate[22] = {
-    [PNSubscribeOperation] = @"/subscribe/{sub-key}/{channels}/0/{tt}",
+    [PNSubscribeOperation] = @"/v2/subscribe/{sub-key}/{channels}/0/{tt}",
```

Comment:
@jzucker2 according to docs there should be no _{tt}_ anymore in request and it shouldn't be added during subscription loops.